build.gradle

```diff
@@ -19,7 +19,7 @@ sourceCompatibility = '1.6'
 
 dependencies {
     compile gradleApi()
-    compile 'org.jvnet.localizer:maven-localizer-plugin:1.13'
+    compile 'org.jvnet.localizer:maven-localizer-plugin:1.12'
```

Comment:
do not downgrade the version, the artifact is in http://repo.jenkins-ci.org/releases/ which is the official repo for all Jenkins related artifacts